### PR TITLE
M2 gcs browser empty state

### PR DESCRIPTION
**Pull request by ivannaranjo**
This PR contains changes to show a message in the data grid when there are no items in the current directory or when there's an error while loading the items for the current directory.

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/GcsFileBrowser/GcsFileBrowserWindowControl.xaml [hunk 1]
```diff
@@ -175,5 +174,41 @@
                 </DataGridTemplateColumn>
             </DataGrid.Columns>
         </DataGrid>
+
+        <!-- The message area -->
+
+        <!-- Empty directory message -->
+        <TextBlock Grid.Row="2"
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/GcsFileBrowser/GcsFileBrowserWindowControl.xaml]: The Grid only had 2 rows. Row index 2 is out of bounds.

File: GoogleCloudExtension/GoogleCloudExtension/GcsFileBrowser/GcsFileBrowserWindowControl.xaml [hunk 2]
```diff
@@ -175,5 +174,41 @@
...
+                    HorizontalAlignment="Center"
+                    Visibility="{Binding CurrentState.IsError, Converter={StaticResource commonVisibilityConverter}}"
+                    VerticalAlignment="Center">
+            <StackPanel Orientation="Horizontal">
+                <Image Source="{mb:ImageResource Utils/Resources/ic_error_red_24px.png }"
+                       VerticalAlignment="Center"
+                       Margin="0,0,5,0"
+                       Width="24px"
+                       Height="24px" />
+                <TextBlock Grid.Column="1"
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/GcsFileBrowser/GcsFileBrowserWindowControl.xaml]: This TextBlock is in a StackPanel. Grid.Column is not used.

**Comment by ivannaranjo** [comment]
I hadn't requested any review yet, I was going to request the review _after_ the previous PR was in.

**Comment by ivannaranjo** [comment]
Because the comments are small I will make the changes and merge in the PR.

**Comment by ivannaranjo** [comment]
Thank you for the preemtive review.

File: GoogleCloudExtension/GoogleCloudExtension/GcsFileBrowser/GcsFileBrowserWindowControl.xaml (hunk 1 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/GcsFileBrowser/GcsFileBrowserWindowControl.xaml]: True, the Grid layout is very forgiving. Fixed.

File: GoogleCloudExtension/GoogleCloudExtension/GcsFileBrowser/GcsFileBrowserWindowControl.xaml (hunk 2 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/GcsFileBrowser/GcsFileBrowserWindowControl.xaml]: True, done.

**Pull request merged**